src/overlays/actors/ovl_Bg_Spot08_Bakudankabe/z_bg_spot08_bakudankabe.c

```diff
@@ -32,8 +33,6 @@ const ActorInit Bg_Spot08_Bakudankabe_InitVars = {
 };
 
 extern Gfx D_0500A880[];
```

Comment:
`gameplay_field_keep` is in, so it should be possible to replace/remove this symbol and extern.